### PR TITLE
Make sure replicator gets started after service creation

### PR DESCRIPTION
**Pull request by Dany9966**

### CODE REVIEW

File: coriolis/providers/replicator.py [hunk 1]
```diff
@@ -719,6 +719,7 @@ def _setup_replicator(self, ssh):
         certs = self._setup_certificates(ssh, args)
         self._exec_replicator(
             ssh, args["port"], certs["remote"], REPLICATOR_STATE)
+        self.start()
```
Comment (aznashwan) [inline coriolis/providers/replicator.py:722]: Is this idempotent? (i.e. are we sure it won't error out if we don't run this if the service is already started?)

Comment (Dany9966) [reply coriolis/providers/replicator.py:722]: > Is this idempotent? (i.e. are we sure it won't error out if we don't run this if the service is already started?)

I tested it: on first replica run, the service is already started before this call, and no error occurs on another start call.

Comment (aznashwan) [reply coriolis/providers/replicator.py:722]: Great, thanks for the confirmation!

**Comment by aznashwan** [comment]
LGTM. Merging. Thanks!

**Pull request merged**